libs/feature/record/src/lib/record-metadata/record-metadata.component.html

```diff
@@ -1,22 +1,41 @@
 <div id="about" class="container-lg mx-auto mb-16" style="margin-top: 60px">
-  <gn-ui-metadata-info
-    class="block"
-    *ngIf="(facade.isPresent$ | async) === false"
-    [metadata]="{
-      title: 'Loading...'
-    }"
-    [incomplete]="true"
-  >
-  </gn-ui-metadata-info>
-  <gn-ui-metadata-info
-    class="block"
-    *ngIf="(facade.isPresent$ | async) === true"
-    [metadata]="facade.metadata$ | async"
-    [source]="facade.source$ | async"
-    [incomplete]="facade.isIncomplete$ | async"
-    [landingPages]="facade.landingPageLinks$ | async"
-  >
-  </gn-ui-metadata-info>
+  <div class="grid grid-cols-3 gap-6">
+    <div class="col-span-2">
+      <gn-ui-metadata-info
+        class="block"
+        *ngIf="(facade.isPresent$ | async) === false"
+        [metadata]="{
+          title: 'Loading...'
+        }"
+        [incomplete]="true"
+      >
+      </gn-ui-metadata-info>
+      <gn-ui-metadata-info
+        class="block"
+        *ngIf="(facade.isPresent$ | async) === true"
+        [metadata]="facade.metadata$ | async"
+        [incomplete]="facade.isIncomplete$ | async"
+        [landingPages]="facade.landingPageLinks$ | async"
+      >
+      </gn-ui-metadata-info>
+    </div>
+    <div>
```

Comment:
`(facade.isPresent$ | async) === true` can be put here, it avoids to multiply this check.
BTW, is it the same as `(facade.metadata$ | async)` ? If yes, you could use it to create an alias and avoid complexity with too many `async` pipes in nested markup.